### PR TITLE
[Fortune Obadiah] Update Readme  With Assigned Task

### PR DESCRIPTION
**Pull request by triumphant-fortune**
# What this PR does
- I've included the contributor task assigned to me, along with the relevant research links.
- Additionally, I've uploaded the first two blog posts I created as part of the task.

### CODE REVIEW

**Comment by codewithmide** [comment]
Well done @triumphant-fortune

File: 2023/Fortune_Obadiah/README.md [hunk 1]
```diff
@@ -12,9 +12,96 @@ I am enthusiastic and fully prepared to roll up my sleeves and immerse myself in
...
+
+[Wagtail User Guide](https://guide.wagtail.org/en-latest/)
+
+
+### Accessibility Features Documentation
+
+- [Accessibility features documentation](#accessibility-features-documentation)
+
+## First Blog Post(Research)
+[My First Journey In Open Source And Git: Overcoming Fear And Embracing Growth](https://medium.com/@fortuneweb3info/navigating-my-first-journey-in-open-source-and-git-overcoming-fear-and-embracing-growth-7f1167f5a6d8)
```
Comment (thibaudcolas) [inline 2023/Fortune_Obadiah/README.md:104]: Nice one!

File: 2023/Fortune_Obadiah/README.md [hunk 2]
```diff
@@ -12,9 +12,96 @@ I am enthusiastic and fully prepared to roll up my sleeves and immerse myself in
...
+
+## Second  Blog Post(Acessibility Features Docuentation)
+[Unlocking Digital Inclusion: The Quest for Accessibility](https://fortuneweb3info.medium.com/unlocking-digital-inclusion-the-quest-for-accessibility-a0a36e99e4b6)
```
Comment (thibaudcolas) [inline 2023/Fortune_Obadiah/README.md:107]: Your first image doesn’t have alt text, which is a big no-no for something like this. Please familiarise yourself with accessibility requirements when writing content on the web.

Aside from your point about Medium, this is all very generic information, which I don’t think does much to _Demonstrate your understanding based on past experiences_.

Comment (triumphant-fortune) [reply 2023/Fortune_Obadiah/README.md:107]: Thank you for the corrections will do a retouch on it 👐🏾  @thibaudcolas

Comment (triumphant-fortune) [reply 2023/Fortune_Obadiah/README.md:107]: I have added an alt text to blog 1 @thibaudcolas

**Pull request merged**